sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxGatewayStoreModel.java

```diff
@@ -85,6 +95,22 @@ public RxGatewayStoreModel(
         this.sessionContainer = sessionContainer;
     }
 
+    void setGatewayServiceConfigurationReader(GatewayServiceConfigurationReader gatewayServiceConfigurationReader) {
+        this.gatewayServiceConfigurationReader = gatewayServiceConfigurationReader;
+    }
+
+    public void setPartitionKeyRangeCache(RxPartitionKeyRangeCache partitionKeyRangeCache) {
+        this.partitionKeyRangeCache = partitionKeyRangeCache;
+    }
+
+    public void setUseMultipleWriteLocations(boolean useMultipleWriteLocations) {
+        this.useMultipleWriteLocations = useMultipleWriteLocations;
+    }
+
+    public void setCollectionCache(RxClientCollectionCache collectionCache) {
```

Comment:
ditto

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/25555#discussion_r754658494